s3backgrounddelete/s3backgrounddelete/config/s3_background_delete_config.yaml.sample

```diff
@@ -67,3 +67,12 @@ leakconfig:                                  # Section for object leak config
    leak_processing_delay_in_mins: 15                          # Time delay in mins after which probable delete entry is procesed for leak.
    version_processing_delay_in_mins: 5                        # Time delay in mins after which object version is processed
    cleanup_enabled: False                                     # Flag for cleanup during S3 instance non-active.
+
+background_delete_account:
```

Comment:
add comment sAYING DO NOT CHANGE KEY NAME. as it will impact creation of bg delete account.